traffic_ops/traffic_ops_golang/tenant/tenancy.go

```diff
@@ -45,13 +45,9 @@ func (dsInfo DeliveryServiceTenantInfo) IsTenantAuthorized(user auth.CurrentUser
 // returns tenant information for a deliveryservice
 func GetDeliveryServiceTenantInfo(xmlId string, db *sqlx.DB) (*DeliveryServiceTenantInfo, error) {
 	ds := DeliveryServiceTenantInfo{}
-	tenantInfoStmt, err := db.Preparex("SELECT xml_id,tenant_id FROM deliveryservice where xml_id = $1")
-	if err != nil {
-		log.Errorf("deliveryservice sql query error, xmlId '%s': %v\n", xmlId, err)
-		return nil, err
-	}
+	query := "SELECT xml_id,tenant_id FROM deliveryservice where xml_id = $1"
 
-	err = tenantInfoStmt.Get(&ds, xmlId)
+	err := db.Get(&ds,query,xmlId)
```

Comment:
I know that's the way it was previously,  but that should be `XMLID` because XML and ID are both acronyms..

Reply:
I think `xmlID` is preferable. Acronyms should be consistent case, but all lower is acceptable, and local variables (which are unexported) should start with lower case, per https://github.com/golang/go/wiki/CodeReviewComments#initialisms https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps